test/test_helper.rb

```diff
@@ -2,3 +2,79 @@
 require 'tyrant'
 
 require 'minitest/autorun'
+
+require 'warden'
+require 'rack-minitest/test'
+
+# Use Warden spec helper to simulate Warden
+# https://github.com/hassox/warden/blob/master/spec/helpers/request_helper.rb
+module Warden::Spec
```

Comment:
I'll write an issue to know how they want to manage this (probably don't want to copy the whole class) and propose my warden method too. We should like into Devise test to see how they manage that, but Devise code is frightening. So huge!